main.py

```diff
@@ -1,27 +1,48 @@
+"""Server file
+Attributes:
+    app (fastapi.applications.FastAPI): Fast API app
+"""
 import os
 import dotenv
 import uvicorn
+from fastapi import FastAPI
 from pydantic import BaseModel
-from typing import Union, Optional
-from fastapi import FastAPI, Response
+from core.classifiers import BOWSubclassPredictor, BERTSubclassPredictor
```

Comment:
These modules should be imported after loading environment variables, for they may depend on some of those variables. Please move them after the `dotenv.load_dotenv()` statement.